apps/server/src/modules/video-conference/bbb/bbb.service.ts

```diff
@@ -40,21 +41,23 @@ export class BBBService {
 		const conf = { headers: { 'Content-Type': 'application/xml' } };
 		const data = this.getBbbRequestConfig(this.presentationUrl);
 		const observable: Observable<AxiosResponse<string>> = this.httpService.post(url, data, conf);
+
 		return firstValueFrom(observable)
 			.then((resp: AxiosResponse<string>) => {
 				const bbbResp = this.converterUtil.xml2object<BBBResponse<BBBCreateResponse> | BBBResponse<BBBBaseResponse>>(
 					resp.data
 				);
 				if (bbbResp.response.returncode !== BBBStatus.SUCCESS) {
-					throw new InternalServerErrorException(bbbResp.response.messageKey, bbbResp.response.message);
+					throw new InternalServerErrorException(`${bbbResp.response.messageKey}: ${bbbResp.response.message}`);
 				}
 				return bbbResp as BBBResponse<BBBCreateResponse>;
 			})
 			.catch((error) => {
-				throw new InternalServerErrorException(error);
+				throw new InternalServerErrorException(ErrorUtils.convertUnknownError(error, 'BBBService:create'));
```

Comment:
Here you hand in the HttpExceptionOptions as first argument which is not correct. The first argument should be a string. 

Also on several places below in this file.

Reply:
Habt ihr auch getestet, ob dann nicht wieder der Hinweis geloggt wird, dass das deprecated ist? Würde ich von ausgehen. Das war doch der Grund für das Ticket.

Aber selbst wenn nicht, finde ich es verwirrend die Options als erstes Argument mitzugeben, wenn sie doch ausdrücklich als zweites spezifiziert sind.

Reply:
Ja das funktioniert. Wir haben getestet. Da wir immer unbekannte Fehler zu ein uns bekannten Format umwandle und ins *cause* packen wollen. Und wir mit NestExceptions arbeiten ist das aus unserer Sicht richtige Vorgehen, die in der Zukunft durch LoggableException aufgelöst oder weiterentwickelt wird.

Reply:
Hab gerade nachgeguckt und der Log wird dadurch in der Tat nicht ausgelöst, weil da nur auf Error gecheckt wird: https://github.com/nestjs/nest/blob/v9.2.1/packages/common/exceptions/http.exception.ts#L87

Trotzdem finde ich es absurd, jetzt die Options als ersten Parameter mitzugeben. In dem Logeintrag steht doch ausdrücklich: `You should use the "options" parameter instead: new HttpException("message", 400, { cause: new Error("Some Error") }) ')` So ist es auch in der Nest-Doku gezeigt: https://docs.nestjs.com/exception-filters#built-in-http-exceptions 

Warum wollt ihr das nicht beachten, wie das von Nest empfohlen wird?

Reply:
Weil wir verschiedene Konstellationen haben und nicht immer im Code als erster Parameter String haben. Ich kann dir morgen noch mal meine Testfälle zeigen.

Reply:
Es tut auch nicht weh immer explicit den string als ersten Parameter rein zu geben. Dann muss dieser aber immer dem default string des jeweiligen Errors entsprechen.

Reply:
Der erste Parameter ist die Message. Da können wir doch reinschreiben, was wir wollen. 

Und wenn man will, dass die Message gleich dem Namen der Exception ist, dann kann man null als ersten Parameter übergeben.

Reply:
Dann schreibe ich dort erstmal überall null mit rein.